backend/package.json

```diff
@@ -12,7 +12,8 @@
   "author": "Marvin Voss, Maximilian Koenig",
   "license": "ISC",
   "devDependencies": {
-    "@types/bun": "latest"
+    "@types/bun": "latest",
+    "bun-types": "^1.1.20"
```

Comment:
Why you installed bun-types additionally? The [NPM](https://www.npmjs.com/package/bun-types) page writes that these two are the same. @types/bun is an alias but the documentation writes to install that. So you could remove bun-types again?

Reply:
This is a left over from testing on make the tests work on the gh pipe. So yes, can be removed.